sway/tree/view.c

```diff
@@ -1260,6 +1261,19 @@ bool view_is_transient_for(struct sway_view *child,
 		child->impl->is_transient_for(child, ancestor);
 }
 
+bool view_can_tear(struct sway_view *view) {
+	switch(view->tearing_mode) {
```

Comment:
Style: missing space between `switch` and `(`

Reply:
Fixed.